lib/matplotlib/legend.py

```diff
@@ -180,6 +180,8 @@ def __init__(self, parent, handles, labels,
                  title=None,  # set a title for the legend
 
                  framealpha=None,  # set frame alpha
+                 edgecolor=None,  # frame patch edgecolor
```

Comment:
Why?  They are all documented as kwargs, not positional arguments, and in the documentation their order differs from their originally random order in the signature.